### DISCUSSION TITLE
Use without internet

### DISCUSSION THREAD

**Discussion by ivan-pavan** [globaleaks/globaleaks-whistleblowing-software#3874] [Q&A]
Hi everyone, I'm implementing the software on an intranet server but when I close the internet connection access to the platform no longer works. Even from clients that are not connected to the internet I cannot reach the platform. Is there a way to use Globaleaks without an internet connection?

**Comment by evilaliv3** [comment]
Hello @ivan-pavan 

GlobaLeaks as any whistleblowing platform in relation to the EU Directive on Whistleblowing should be implemeted in the open internet to achieve best result and in best compliance but nothing technically prevents that it could work also in side an intranet.

Which errors are you encountering?

**Reply by ivan-pavan** [reply]
The Globaleaks server is installed on a VM with Debian 12 with two network cards: network card 1 with internet access and network card 2 with access to the production network.
Not all PCs on the production network have internet access.
My goal is to allow all PCs on the production network (both those connected to the internet and those not connected to the internet) to access the pages of the Globaleaks server via the IP address of network card 2 (access to the production network).
As long as network card 1 (internet access) of the Globaleaks server is active, PCs on the production network with internet access can connect to the Globaleaks server using the IP address associated with network card 2 (production network access) , PCs without internet access will not connect to the Globaleaks server.
If I deactivate network card 1 (internet access) of the Globaleaks server, no PC on the production network connects to the Globaleaks server page (it works on neither of the two network cards).
In your opinion, is there a way to use the Globaleaks service without the Globaleaks server and without the PCs being connected to the internet?
